config/environments/production.rb

```diff
@@ -96,14 +96,4 @@
   config.action_mailer.perform_deliveries = true
 
   config.action_mailer.default_url_options = { host: ENV['APP_URL'] }
-
-  ActionMailer::Base.smtp_settings = {
```

Comment:
We need to figure out a way to parameterize this that doesn't include removing it completely, perhaps we can get a default that is just all of these things as ENV's, that way this still works on Heroku.